### DISCUSSION TITLE
Where do I report missing picons?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#40182] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

     Hi

     I downloaded snp-full.220x132-190x102.light.on.transparent (2022-09-01--15-47-57) in the openATV 7.1 Plugin Manager, unfortunately there are still a few picons missing that I could deliver...
     But they still need some editing.

     Where do I report missing picons?

     I added the setting later... there are not many picons missing for the setting. (See attached file)
     Above all, missing Swiss picons can all be found here https://www.sunrise.ch/en/internet-tv/channel-list overview they would simply have to be adjusted....
     Downloaded the most important ones (see attached file Sunrise Picon)

     If every transmitter really had a cleanly adapted picon, that would be awesome.
[Picon Sunrise.zip](https://github.com/community/community/files/10097320/Picon.Sunrise.zip)
[setting-icam-mixed-sat-cable-upc-innerschweiz.zip](https://github.com/community/community/files/10097321/setting-icam-mixed-sat-cable-upc-innerschweiz.zip)

**Comment by jfullstackdev** [comment]
if this is the project, you can report it there by creating an issue

https://github.com/picons/picons

**Comment by ghost** [comment]
I wanted to do that but don't know where and how?

**Comment by jfullstackdev** [comment]
go here

https://github.com/picons/picons/issues

click `New issue`

![image](https://user-images.githubusercontent.com/47092464/204119251-90070bc3-c010-4b50-b4f1-a21e43d178a7.png)

you will be prompted with their guidelines on proper creation of issue

**Comment by ghost** [comment, marked as answer]
Feedback on the last update, all picons from the provider Sunrise are available. Everything perfect :-)

For info on Sat 19.2 the channel Schau TV is now called Kurier TV and needs a new picon...

**Reply by ghost** [reply]
![kurier-tv-at](https://user-images.githubusercontent.com/35585833/218225453-cdcbc672-878b-4aed-a9c8-e38d2fcc8c66.svg)
